### ISSUE TITLE
Loading the model:

### ISSUE THREAD

**Issue by mars11121** [anchen1011/toflow#22]
To reproduce the results of the paper on video denoising, I want to load the pre-trained model to evaluate on the test set of Vimeo-90 dataset. But when I reload the model using torch7, I have encountered the following problem: 

Warning: Failed to load function from bytecode: binary string: not a precompiled chunkWarning: Failed to load function from bytecode: [string ""]:1: unexpected symbol near char(5)/home/mars/torch/install/bin/lua: /home/mars/torch/install/share/lua/5.2/torch/File.lua:375: unknown object
stack traceback:
	[C]: in function 'error'
	/home/mars/torch/install/share/lua/5.2/torch/File.lua:375: in function 'readObject'
	/home/mars/torch/install/share/lua/5.2/torch/File.lua:307: in function 'readObject'
	/home/mars/torch/install/share/lua/5.2/torch/File.lua:369: in function 'readObject'
	/home/mars/torch/install/share/lua/5.2/nn/Module.lua:192: in function 'read'
	...rs/torch/install/share/lua/5.2/nn/BatchNormalization.lua:185: in function 'read'
	/home/mars/torch/install/share/lua/5.2/torch/File.lua:351: in function 'readObject'
	/home/mars/torch/install/share/lua/5.2/torch/File.lua:369: in function 'readObject'
	/home/mars/torch/install/share/lua/5.2/torch/File.lua:369: in function 'readObject'
	/home/mars/torch/install/share/lua/5.2/nn/Module.lua:192: in function 'read'
	...
	/home/mars/torch/install/share/lua/5.2/nn/Module.lua:192: in function 'read'
	/home/mars/torch/install/share/lua/5.2/torch/File.lua:351: in function 'readObject'
	/home/mars/torch/install/share/lua/5.2/torch/File.lua:369: in function 'readObject'
	/home/mars/torch/install/share/lua/5.2/torch/File.lua:369: in function 'readObject'
	/home/mars/torch/install/share/lua/5.2/nn/Module.lua:192: in function 'read'
	/home/mars/torch/install/share/lua/5.2/torch/File.lua:351: in function 'readObject'
	/home/mars/torch/install/share/lua/5.2/torch/File.lua:409: in function 'load'
	demo.lua:65: in main chunk
	[C]: in function 'dofile'
	...mars/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: in ?
Could you help me check if the pre-trained model has any problem? @anchen1011
